ee/scripts/schema/db/init_dbs/postgresql/1.20.0/1.20.0.sql

```diff
@@ -61,6 +61,51 @@ ALTER TABLE IF EXISTS public.users
       ]
     }'::jsonb;
 
+CREATE TABLE IF NOT EXISTS spots
+(
+    spot_id    BIGINT NOT NULL PRIMARY KEY,
+    name       TEXT NOT NULL,
+    user_id    BIGINT NOT NULL REFERENCES public.users (user_id) ON DELETE CASCADE,
+    tenant_id  BIGINT NOT NULL,
+    duration   INT NOT NULL,
+    crop       INT[],
+    comments   TEXT[],
+    status     TEXT DEFAULT 'pending',
+    created_at timestamp NOT NULL,
```

Comment:
you can change it to 
`created_at timestamp without time zone NOT NULL DEFAULT (now() at time zone 'utc')`